src/AllensCausalModel/CausalModels.jl

```diff
@@ -0,0 +1,57 @@
+module CausalModels
+using Distributions
+using Statistics
+export ExogenousVariable, EndogenousVariable, intervene, randomsample, prob
+	#represent as function with id
+	function ExogenousVariable(name,dist)
```

Comment:
Give this function a docstring.  Docstrings are strings before functions and structs and types
For instance:

```julia
"maps `x` to `sqrt(x)`"
f(x) = sqrt(x)
```